### PR TITLE
allow passing custom printers for pretier results

### PR DESCRIPTION
**Pull request by nadav-ab**
allow passing printers in order to get better print output

### CODE REVIEW

File: packages/mostly-equal/package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "main": "dist/cjs/index.js",
   "module": "dist/esm/index.js",
   "scripts": {
-    "test": "mocha \"dist/test/**/*.unit.js\""
```
Comment (daomry) [inline packages/mostly-equal/package.json:7]: can we drop the scripts here?

File: packages/mostly-equal/src/mostly-equal-chai-plugin.ts [hunk 2]
```diff
@@ -1,16 +1,47 @@
...
+    maxDepth: 10,
+    replacers: [],
+};
+
+export const setGlobalOptions = (maxDepth: number, replacers?: Replacer[]) => {
+    globalOptions.maxDepth = maxDepth;
+    globalOptions.replacers = replacers;
+};
+
+export const tempSetGlobalOptions = (
```
Comment (daomry) [inline packages/mostly-equal/src/mostly-equal-chai-plugin.ts:22]: I hate this API, but if you're going to keep it, please rename to something helpful
(setSuiteOptions or something)

**Review by daomry — CHANGES_REQUESTED** [review]

File: packages/mostly-equal/package.json (hunk 1 above)
Comment (nadav-ab) [reply packages/mostly-equal/package.json:7]: why? it helps when debugging

File: packages/mostly-equal/src/mostly-equal-chai-plugin.ts (hunk 2 above)
Comment (nadav-ab) [reply packages/mostly-equal/src/mostly-equal-chai-plugin.ts:22]: done

File: packages/mostly-equal/src/test/mostly-equal.unit.ts [hunk 3]
```diff
@@ -290,4 +290,24 @@ describe('mostly equal', () => {
             expect(values).to.eql(expectedMatched);
         });
     });
+
+    describe('global options', () => {
```
Comment (daomry) [inline packages/mostly-equal/src/test/mostly-equal.unit.ts:294]: setOptionsForSuite ?

**Review by daomry — CHANGES_REQUESTED** [review]

**Pull request merged**